du.md

```diff
@@ -0,0 +1,61 @@
+---
+---
+
+du
+-------
+
+The du (disk usage) command shows you the sizes of directory trees including all of their contents and sizes of individual files.
+This makes it usefull for taking down particular directories that are hogging disk space. 
+
+~~~ bash
+$ du
+92	.
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### 'du -h'
+~~~ bash
+$ du -h
```

Comment:
I think this is a particularly useless example. What about something like this?

    $ mkdir test
    $ yes > test/bigfile.txt
    $ du -h
    51M    ./test
    51M    .